Automattic/wp-calypso
# Stats: change /calypso/images -> calypso/assets/images to allow correct resolving of assets

#### Proposed Changes
The PR changes all assets referred by the Stats App from `/calypso/images/...` to `calypso/assets/images/...` which allows correct resolving by the loaders and also has benefits for cache busting. Assets in `/static` are retained because we are not sure where they are also referenced.

Also a note for the team: in the future, please do not add more assets to `/static`, the correct location is `client/assets/`.

More information: https://github.com/Automattic/wp-calypso/blob/954e16bee68d10e4031f8d8fb22b3546abf5c687/static/images/README.md

#### Testing Instructions
* Open a site with Jetpack plugin
* Open `/wp-admin/admin.php?page=jetpack#/traffic?term=jetpack%20stats`
* Expand the Jetpack Stats section
* Open browser console and run `document.querySelector('#jp-settings-site-stats').querySelectorAll('.jp-form-fieldset')[1].setAttribute('style','color:red')`
* Turn on the new Stats experience (color red toggle)
* Open `/wp-admin/admin.php?page=stats`
* Ensure the page looks reasonable, no assets 404

* Checkout the branch run `yarn start`
* Open `http://calypso.localhost:3000/` for Stats pages
* Ensure no assets 404

<img width="623" alt="image" src="https://user-images.githubusercontent.com/1425433/203453009-5184f2b8-482c-4193-b603-d62a071a4dbd.png">

<img width="495" alt="image" src="https://user-images.githubusercontent.com/1425433/203453143-f067cc78-77a0-4d6b-8e16-635c9459040c.png">


#### Pre-merge Checklist





Related to #
